ci/cpu/prebuild.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+#Upload cuspatial once per CUDA
```

Comment:
Only upload on CUDA 9.2 so it uploads on both Python 3.6 and 3.7
```suggestion
#Upload cuspatial once per PYTHON
```